apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.IntegrationTests/JobsTest.cs

```diff
@@ -67,5 +69,210 @@ public void ListJobs_FullProjection()
 
             Assert.NotNull(jobFound.Resource.Configuration);
         }
+
+        [Fact]
+        public void ListJobs_JobLabels()
+        {
+            var client = BigQueryClient.Create(_fixture.ProjectId);
+
+            var table = client.GetTable(_fixture.ProjectId, _fixture.DatasetId, _fixture.HighScoreTableId);
+            var jobToFind = client.CreateQueryJob($"SELECT * FROM {table}", null, new QueryOptions() { Labels = JobLabels});
+
+            // Find the job after listing with full projection.
+            var jobFound = (from job in client.ListJobs(new ListJobsOptions() { Projection = ProjectionEnum.Full })
+                            where job.Reference.JobId == jobToFind.Reference.JobId
+                            select job).Single();
+
+            VerifyJobLabels(jobFound.Resource.Configuration.Labels);
+        }
+
+        [Fact]
+        public void GetJob()
+        {
+            var client = BigQueryClient.Create(_fixture.ProjectId);
+            var table = client.GetTable(_fixture.ProjectId, _fixture.DatasetId, _fixture.HighScoreTableId);
+            var jobToFind = client.CreateQueryJob($"SELECT * FROM {table}", parameters: null);
+
+            var jobFound = client.GetJob(jobToFind.Reference);
+
+            Assert.Equal(jobToFind.Reference.JobId, jobFound.Reference.JobId);
+        }
+
+        [Fact]
+        public void GetJob_JobLabels()
+        {
+            var client = BigQueryClient.Create(_fixture.ProjectId);
+            var table = client.GetTable(_fixture.ProjectId, _fixture.DatasetId, _fixture.HighScoreTableId);
+            var options = new QueryOptions() { Labels = JobLabels };
```

Comment:
Just as a matter of style, I'd suggest omitting the `()` when calling a parameterless constructor with an object initializer. It doesn't make any difference here, but I think it's good to have a consistent style in the snippets, and the brevity is appealing :)

Reply:
Done, here and elsewhere.